install_files/ansible-base/roles/tails-config/tasks/install_shell_extension.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: Create the SecureDrop GNOME Shell Extension directory
+  become: yes
+  file:
+    path: "{{ tails_config_live_dotfiles }}/.local/share/gnome-shell/extensions/securedrop@securedrop.freedom.press"
```

Comment:
Files here are symlinked into the home directory at the start of a session - so this directory and associated files are not available when the `gnome-extensions enable` command runs below. It looks like the desktop icon tasks work around this by initially installing the icons both into `~amnesia/.local/share/applications` and into `/live/persistence/TailsData_unlocked/dotfiles/.local/share/applications` so that they're available both on the first run and on subsequent sessions. Something similar might work here.